frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewListItem.tsx

```diff
@@ -56,6 +56,7 @@ const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
                   innerRadius={8}
                   outerRadius={12}
                   title={availableReplicas}
+                  subTitle={''}
```

Comment:
Instead of passing `subTitle` as empty string here from the caller, I suggest better make this optional prop to have a default value set to empty string in PodStatus Component.

Reply:
thanks :D
updated PR